app/src/modules/settings/routes/presets/item.vue

```diff
@@ -129,9 +129,9 @@
 	</component>
 </template>
 
-<script lang="ts">
+<script setup lang="ts">
```

Comment:
Just out of curiosity, what are the benefits/arguments for using this `<script setup` form over the `defineComponent` form? the latter feels more declarative to me but as a relative newcomer to Vue having all the different forms can feel a bit like a mess in the ecosystem sometimes 😓

Reply:
`script setup` is funnily more declarative as with it you define props and emits way more intuitive and the main benefit of using script setup is way less overhead. When you write components using `defineComponent` you have to mannually expose each variable and function to the template which comes annoying really quickly where as on `script setup`, everything is public by default.